nibabel/nifti1.py

```diff
@@ -1764,7 +1764,20 @@ def __init__(self, dataobj, affine, header=None,
         if header is None and affine is not None:
             self._affine2header()
     # Copy docstring
-    __init__.doc = analyze.AnalyzeImage.__init__.__doc__
+    __init__.__doc__ = analyze.AnalyzeImage.__init__.__doc__ + '''
+    Notes
+    -----
+
+    If both a ``header`` and an ``affine`` are specified, and the ``affine``
```

Comment:
I think you want single backticks for the parameters `header` and `affine`.

Reply:
I think it's a single backtick for markdown, but double backticks for restructured text - this renders fine as it is.

Reply:
Single backticks in rst gives the default markup, which, in the case of the numpy docstring renderer, has the right markup for referring to parameters.   So, single backticks to refer to parameters, double backticks for typewriter font ...

Reply:
Aah ok - I am not all that familiar with numpydoc - will fix it now!